dali/operators/image/color/brightness_contrast.cu

```diff
@@ -29,6 +29,8 @@ DALI_REGISTER_OPERATOR(BrightnessContrast, BrightnessContrastGpu, GPU)
 
 bool BrightnessContrastGpu::SetupImpl(std::vector<OutputDesc> &output_desc,
                                       const workspace_t<GPUBackend> &ws) {
+  auto curr_batch_size = ws.GetInputBatchSize(0);
```

Comment:
It makes a false impression that it does anything meaningful for GPU, while it s always `kernel_manager_.Resize(1, 1);`.

Reply:
@mzient ?

Reply:
1. It should be KMgrResize - if we indeed need a function for that.
2. It should resize to max_batch_size_, not to current - otherwise you'll keep creating and destroying kernel instances - therefore, it can just stay in the constructor.